src/generic/eigen_solver.h

```diff
@@ -238,6 +252,14 @@ namespace oomph
                             Vector<std::complex<double>>& eigenvalue,
                             Vector<DoubleVector>& eigenvector);
 
+    void solve_adjoint_eigenproblem(Problem* const& problem_pt,
+                                    const int& n_eval,
+                                    Vector<std::complex<double>>& eigenvalue,
+                                    Vector<DoubleVector>& eigenvector)
+    {
+      BrokenCopy::broken_assign("LAPACK_QZ");
```

Comment:
This is the wrong way to break it! We're not breaking an assignment operator. Please put a customised error in there.

Reply:
I'm confused. did you update the code? As far as I can tell it still says
```
    void solve_adjoint_eigenproblem(Problem* const& problem_pt,
                                    const int& n_eval,
                                    Vector<std::complex<double>>& eigenvalue,
                                    Vector<DoubleVector>& eigenvector)
    {
      BrokenCopy::broken_assign("ARPACK");
    }
```

Reply:
Sorry, found it:
```
      throw OomphLibError("solve_adjoint_eigenproblem is not currently "
                          "implemented for LAPACK_QZ.",
                          OOMPH_CURRENT_FUNCTION,
                          OOMPH_EXCEPTION_LOCATION);
```
Strange that you provide two separate strings (next to each other) as an argument to a function that expects single string. Fine if it compiles, I suppose. I'd have built the string on the outside and sorted it in a variable that can then be passed to the error. But whatever works...